### ISSUE TITLE
[Introspection] `?` is not accepted as character in column name

### ISSUE THREAD

**Issue by janpio** [prisma/prisma2#1082]
Error:
```
error: Unexpected token. Expected one of: field type.
  -->  schema.prisma:2
   | 
 1 | model data {
 2 |   bruises?                 String?
   |          ^ Unexpected token.
 3 |   cap-color                String?
   | 
```
(https://github.com/prisma/introspection-engine-output/pull/5/files#diff-c85970a5d8c1859c01314cc862153adc)

Model:
```prisma
model data {
  bruises?                 String?
  cap-color                String?
  cap-shape                String?
  cap-surface              String?
  class                    String?
  gill-atachment           String?
  gill-color               String?
  gill-size                String?
  gill-spacing             String?
  habitat                  String?
  odor                     String?
  population               String?
  ring-number              String?
  ring-type                String?
  spore-print-color        String?
  stalk-root               String?
  stalk-shape              String?
  stalk-surface-above-ring String?
  stalk-surface-below-ring String?
  veil-color               String?
  veil-type                String?
}
```
(https://github.com/prisma/introspection-engine-output/pull/5/files#diff-8c2fedbf73a9e2c7342e91924dfcc10d)

Original database:
```
CREATE TABLE `data` (
	`class` VARCHAR(255) NULL DEFAULT NULL,
	`cap-shape` VARCHAR(255) NULL DEFAULT NULL,
	`cap-surface` VARCHAR(255) NULL DEFAULT NULL,
	`cap-color` VARCHAR(255) NULL DEFAULT NULL,
	`bruises?` VARCHAR(255) NULL DEFAULT NULL,
	`odor` VARCHAR(255) NULL DEFAULT NULL,
	`gill-atachment` VARCHAR(255) NULL DEFAULT NULL,
	`gill-spacing` VARCHAR(255) NULL DEFAULT NULL,
	`gill-size` VARCHAR(255) NULL DEFAULT NULL,
	`gill-color` VARCHAR(255) NULL DEFAULT NULL,
	`stalk-shape` VARCHAR(255) NULL DEFAULT NULL,
	`stalk-root` VARCHAR(255) NULL DEFAULT NULL,
	`stalk-surface-above-ring` VARCHAR(255) NULL DEFAULT NULL,
	`stalk-surface-below-ring` VARCHAR(255) NULL DEFAULT NULL,
	`veil-type` VARCHAR(255) NULL DEFAULT NULL,
	`veil-color` VARCHAR(255) NULL DEFAULT NULL,
	`ring-number` VARCHAR(255) NULL DEFAULT NULL,
	`ring-type` VARCHAR(255) NULL DEFAULT NULL,
	`spore-print-color` VARCHAR(255) NULL DEFAULT NULL,
	`population` VARCHAR(255) NULL DEFAULT NULL,
	`habitat` VARCHAR(255) NULL DEFAULT NULL
)
COLLATE='utf8_general_ci'
ENGINE=InnoDB
;

```

![image](https://user-images.githubusercontent.com/183673/70396973-2b819b80-1a0e-11ea-82b5-b17f6c7de76f.png)

**Comment by pantharshit00** [comment]
I can confirm this bug. A temporary workaround here would be to add the @map directive to those fields and changing the identifier to a valid name.

**Comment by janpio** [comment]
fixed via #1139
